### DISCUSSION TITLE
About cookie consent banner

### DISCUSSION THREAD

**Discussion by Ale-JRD** [nunocoracao/blowfish#568] [Q&A]
Hi nunocoracao, great theme.

I would love to know if there is a way on how to configure or insert a cookie consent banner to comply with GDPR.

**Comment by nunocoracao** [comment]
Hello @Ale-JRD please follow https://github.com/nunocoracao/blowfish/issues/468
This request has already been made so you can vote on the ticket and add a comment there if you would like. 

You can always extend the theme using Hugo overrides and build it yourself for now. More info on that on Hugo Docs
